packages/main/src/components/VariantManagement/ManageViewsDialog.tsx

```diff
@@ -22,12 +22,14 @@ import { cssVarVersionInfoPrefix } from '../../internal/utils.js';
 import { Bar } from '../../webComponents/Bar/index.js';
 import { Button } from '../../webComponents/Button/index.js';
 import { Dialog } from '../../webComponents/Dialog/index.js';
+import type { InputDomRef } from '../../webComponents/index.js';
 import { Icon, Input } from '../../webComponents/index.js';
 import { Table } from '../../webComponents/Table/index.js';
 import { TableColumn } from '../../webComponents/TableColumn/index.js';
 import { FlexBox } from '../FlexBox/index.js';
 import { ManageViewsTableRows } from './ManageViewsTableRows.js';
 import type { VariantItemPropTypes } from './VariantItem.js';
+import type { VariantManagementPropTypes } from './index.js';
```

Comment:
Maybe we should extract this type into a dedicated `types.ts` file? Otherwise we've now cyclic imports